### PR TITLE
implement coordinate-editing on edit view

### LINKED ISSUE #19: allow user to set coordinates on location-edit page

### PR DESCRIPTION
**Pull request by sethvincent**
closes #19

### CODE REVIEW

**Comment by kamicut** [comment]
argh more yarn merge errors

**Comment by kamicut** [comment]
@sethvincent should there be an old location indicator, or a crosshair? It's not immediately intuitive that I should click on the marker and then click on a new location

**Comment by sethvincent** [comment]
oh yes, if the location already has coordinates it should have something like a marker.

maybe it can automatically be in point mode instead of having to click the point button first.

i'll get the yarn.lock merge conflict.

**Comment by kamicut** [comment]
@sethvincent this is a location I've edited, and added a location to:

image

It has a marker on the detail page but not on the edit page. Is this intended behavior?

**Comment by kamicut** [comment]
One possible improvement would be to center something with 0,0 coordinates on the centroid of the country we know this instrument is in.

File: app/assets/scripts/components/map-edit.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    });
+
+    this.geocoder = new MapboxGeocoder({ accessToken: mapbox.token });
+
+    this.map.addControl(this.draw, 'top-left');
+    this.map.addControl(this.geocoder, 'bottom-left');
+
+    this.map.on('load', () => {
+      if (!usingDefaultCoordinates) {
+        console.log('first point', this.draw.add({
```
Comment (kamicut) [inline app/assets/scripts/components/map-edit.js]:
```suggestion
```

**Pull request merged**